src/templates/app/user/set_currency.jsx

```diff
@@ -56,7 +60,23 @@ const SetCurrency = () => (
                 </form>
             </div>
 
-            <SeparatorLine show_mobile className='gr-padding-30' sub_class='gr-padding-10' invisible />
+            <div data-show='iom' className='center-text invisible has_currency'>
```

Comment:
will this show it to everyone who has currency or just iom clients? I'm worried the js code will make it visible for non-iom clients as well

Reply:
only for iom, data-show attribute should have a higher importance :)

Reply:
cool, you have tested it?

Reply:
yes i did